package.json

```diff
@@ -14,6 +14,7 @@
   },
   "license": "MIT",
   "devDependencies": {
-    "ecmarkup": "^8.1.0"
+    "@tc39/ecma262-biblio": "latest",
```

Comment:
this should not be using `latest`, it should be using a semver range. v2 of this package has a breaking change compared to v1, for example.